src/app/frontend/deploy/deploy.controller.js

```diff
@@ -131,4 +143,24 @@ export default class DeployController {
   isPortMappingEmpty_(portMapping) {
     return !!portMapping.port && !!portMapping.targetPort;
   }
+
+  /**
+   * Initializes namespace selector.
+   * @param {!angular.$log} $log
+   * @param {!angular.$resource} $resource
+   * @private
+   */
+  initializeNamespaces_($log, $resource) {
+    /** @type {!angular.Resource<!backendApi.NamespaceList>} */
+    let resource = $resource('/api/namespace');
```

Comment:
In this file we have two patterns of using $resource now. The other is in line 61. Can you unify approaches so that both styles are the same? 

I mean. In the line 61 we create $resource('foo') object and then use it, and here we create $resource('foo') on demand. I like your solution more.